Arduino/Arduino_config.py

```diff
@@ -0,0 +1,76 @@
+import requests
+
+
+URL = "http://call-flow-testing.appspot.com/"
+
+
+class arduino_Callflow():
+    def get_nodes(self):
+        print(requests.get(URL + "nodes").json())
+
+    def get_node(self, node):
+        print(requests.get(URL + "nodes/{}".format(node)).json())
+
+    def get_Actions_for_batch(self, **kwargs):
+        for key in kwargs.items():
+            if key == "batch":
+                batch = key
+                print(requests.get(URL + "batches/{}".format(batch)).json())
+
+    def get_batches(self):
+        print(requests.get(URL + "batches").json())
+
+    def delete_batches(self, id):
+        print(requests.delete(URL + "batches/{}".format(id)).json())
+
+    def delete_node(self, node):
+        print(requests.delete(URL + "nodes/{}".format(node)).json())
+
+    def create_node(self, **kwargs):
+        for key, value in kwargs.items():
+            if key == "id" and value != "":
+                response = requests.post(URL + "nodes/?", data=kwargs)
```

Comment:
You are looping over a dictionary untill you find the 'id' key and then do a post passing in the dictionary. 
I would be better to do the following:
```
if not kwargs['id']:
    print('error')
    return
response = requests.post(URL + "nodes/?", data=kwargs)
...
```

Reply:
Same for other occurrences.